sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2472,30 +2466,31 @@
       }
     },
     "Twin": {
-      "description": "Twin Representation",
+      "description": "Twin Representation of the device/ module state information. Implicitly created and deleted when the corresponding device/ module identity is created or deleted in IoT Hub.",
       "type": "object",
       "properties": {
         "deviceId": {
-          "description": "The deviceId uniquely identifies the device in the IoT hub's identity registry. A case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars + {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
+          "description": "The unique identifier of the device in the IoT hub's identity registry. It is a case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars, and the following special characters {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
           "type": "string"
         },
         "moduleId": {
-          "description": "Gets and sets the Module Id.",
+          "description": "The unique identifier of the module in the IoT hub's identity registry. It is a case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars, and the following special characters {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
           "type": "string"
         },
         "tags": {
-          "description": "A JSON document read and written by the solution back end. Tags are not visible to device apps.",
+          "description": "A JSON document read and written by the solution back end. They are not visible to device apps.",
```

Comment:
I am not sure why we describe tags like this. Is this not talking about our implementation when it should just talk about its purpose? Maybe we should just say key-value pairs that are not visible to device apps? Is there any other use of tags? Probably we should mention that.

Reply:
This is how the official documents describes tags. I can rephrase it to:
`A collection of key-value pairs read and written by the solution back end. They are not visible to device apps.`
Would that be better?

Reply:
I think the point is to improve this as the original documentation is bad. My question here was what is solution back end? I might be missing context and hence did not get it or we need to describe this better. (probably by asking service team)

Reply:
solution back-end here refers to any application that does service-side updates